frontend/public/components/monitoring/silence-form.tsx

```diff
@@ -214,7 +214,11 @@ const SilenceForm_: React.FC<SilenceFormProps> = ({ defaults, Info, title }) =>
         history.push(`${SilenceResource.plural}/${encodeURIComponent(silenceID)}`);
       })
       .catch((err) => {
-        setError(_.get(err, 'json.error') || err.message || 'Error saving Silence');
+        const errorMessage =
+          typeof _.get(err, 'json') === 'string'
+            ? _.get(err, 'json')
+            : err.message || 'Error saving Silence';
```

Comment:
So, is the error never found in `json.error` here?

Reply:
No because the error is returned as a single string (which is a valid JSON), see: https://github.com/openshift/console/blob/master/frontend/public/co-fetch.ts#L122

Reply:
👍